semp/metricDesc.go

```diff
@@ -169,10 +174,10 @@ var MetricDesc = map[string]Descriptions{
 		"system_redundancy_local_active": NewSemDesc("system_redundancy_local_active", NoSempV2Ready, "Is local node the active messaging node? (0-not active, 1-active).", variableLabelsRedundancy),
 	},
 	"RedundancyHW": {
-		"system_redundancy_role":      NewSemDesc("system_redundancy_role", NoSempV2Ready, "Redundancy role (0=Backup, 1=Primary, 2-Undefined).", variableLabelsRedundancy),
-		"system_redundancy_mode":      NewSemDesc("system_redundancy_mode", NoSempV2Ready, "Redundancy mode (0=Active/Active, 1=Active/Standby).", variableLabelsRedundancy),
-		"system_redundancy_adb_link":  NewSemDesc("system_redundancy_adb_link", NoSempV2Ready, "Is adb link up? (0-no, 1-yes).", variableLabelsRedundancy),
-		"system_redundancy_adb_hello": NewSemDesc("system_redundancy_adb_hello", NoSempV2Ready, "Is adb link connected? (0-no, 1-yes).", variableLabelsRedundancy),
+		"system_redundancy_hw_role":      NewSemDesc("system_redundancy_hw_role", NoSempV2Ready, "Redundancy role (0=Backup, 1=Primary, 2-Undefined).", variableLabelsRedundancyHW),
```

Comment:
Why needs the metric name to be different for hardware? 
This makes it harder to write generic Dashboards.

Reply:
This was because the values for appliances are different (there are no monitoring nodes) and I thought having a metric which referenced monitoring nodes in the help text would look strange.  However, you make a good point about the Dashboards.  Happy to eliminate it and use the original.